Separation of the backup process and the update process (#935)
PrestaShop/autoupgrade: README.md

$ php bin/console update:check <your-admin-dir>
```

> You can create backup with the following command:

Reviewer: I'm looking for a sentence without "you", like

```suggestion
A backup of the shop is created with:
```
